README.md

```diff
@@ -30,173 +26,217 @@ Start by installing the package:
 pip install hayhooks
 ```
 
-The `hayhooks` package ships both the server and the client component, and the client is capable of starting the
-server. From a shell, start the server with:
-
-```console
-$ hayhooks run
-INFO:     Started server process [44782]
-INFO:     Waiting for application startup.
-INFO:     Application startup complete.
-INFO:     Uvicorn running on http://localhost:1416 (Press CTRL+C to quit)
-```
-
 ### Configuration
 
```

Comment:
How about one sentence into what hayhooks is. Simple, clear and to the point. Very important to nail this sentence/paragraph!

Reply:
Good point!

Reply:
I've added a small paragraph